packages/video_player/video_player/pubspec.yaml

```diff
@@ -31,3 +31,9 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+
+
+# FOR TESTING ONLY. DO NOT MERGE.
+dependency_overrides:
```

Comment:
All of the `dependency_overrides` changes should be reverted; this isn't a multi-package PR.